recipes/nionui/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nionui" %}
+{% set version = "0.3.26" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  #url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  #sha256: 85c4a64e6a30be2b601f05680e809712589c6dbb561426d4d5e0f35ef5f97e54
+  url: "https://github.com/nion-software/nionui/archive/{{ version }}.tar.gz"
```

Comment:
We temporary don't use from pypi, since there are missing files in the source distribution, see https://github.com/nion-software/nionui/pull/26.